plugin/PDFView.qml

```diff
@@ -144,6 +144,7 @@ SilicaFlickable {
 
         PinchArea {
             anchors.fill: parent
+            enabled: contextMenuLinks.opacity == 0. && contextMenuGoto.opacity == 0.
```

Comment:
Ok, I'll try to move PDFContextMenu to PDFDocumentPage and see which signals should be exchanged for the menu to interact with the PDFView content (scroll to position…).